packages/mdc-theme/_variables.scss

```diff
@@ -16,20 +16,28 @@
 
 @import "./functions";
 
-/*
-  Main theme colors.
-  If you're a user customizing your color scheme in SASS, these are probably the only variables you need to change.
-*/
-$mdc-theme-primary: #3f51b5 !default; /* Indigo 500 */
-$mdc-theme-accent: #ff4081 !default; /* Pink A200 */
-$mdc-theme-background: #fff !default; /* White */
+//
```

Comment:
Done!

The reason I added the blank lines is because they become necessary in PR #1116. But I'll remove them from this PR (#1112) and add them in #1116 instead.